edx_proctoring/views.py

```diff
@@ -1805,6 +1807,19 @@ def make_review(self, attempt, data, backend=None):
                 {'user': review.reviewed_by, 'attempt_code': attempt_code}
             )
 
+        video_review_link = backend_review.get('payload', {}).get('videoReviewLink')
+        if video_review_link:
+            aes_key_str = backend.get_video_review_aes_key()
+            if aes_key_str:
+                try:
+                    aes_key = codecs.decode(aes_key_str, "hex")
+                    review.encrypted_video_url = encrypt_and_encode(video_review_link.encode("utf-8"), aes_key)
+                except Exception as ex:  # pylint: disable=broad-except
```

Comment:
Do we have any idea what a less broad except might be for this? Does the cryptography library have a base exception class we could use instead? (If we have a similar broad except in idv code, could we check logs to see what more-specific exceptions we've seen from that code?)

Reply:
This is something I added because the encryption of the video url isn't behind a waffle flag, and I didn't want it to break incoming PSI reviews if the AES key that was added was mistakingly invalid (I might have been overly cautious with the addition of this try/catch though).

Once this is tested on stage or prod this try/catch should probably be removed.

Reply:
Actually, if we could only add the AES key on stage first, we could test the encryption of the video url there without worrying about breaking prod. Then there's no longer a need for this error handling (which doesn't exist in IDV anyway)

Reply:
cool, glad I added the keys for prod & stage in separate PRs. That works out well